noir_stdlib/src/hash/poseidon/bn254/perm.nr

```diff
@@ -3,6 +3,7 @@ use crate::hash::poseidon::bn254::consts;
 use crate::hash::poseidon::bn254::permute;
 use crate::hash::poseidon::PoseidonConfig;
 
+#[field(bn254)]
```

Comment:
seems like it would be good if we could add this to the bn254 module instead?

Reply:
I am closing this because I have added an issue for having the field attribute on a module.